Patch to add Solaris to the list of OSes which use the 'prefork'
version of mod-prefork.  This patch has not yet been submitted upstream.

diff --git a/manifests/mod/prefork.pp b/manifests/mod/prefork.pp
index f35551c..1080ce9 100644
--- a/manifests/mod/prefork.pp
+++ b/manifests/mod/prefork.pp
@@ -62,7 +62,7 @@ class apache::mod::prefork (
         }
       }
     }
-    'debian', 'freebsd': {
+    'debian', 'freebsd', 'solaris': {
       ::apache::mpm{ 'prefork':
         apache_version => $_apache_version,
       }
